### ISSUE TITLE
print localhost:9771 in the CLI when a kurtosis run completes

### ISSUE THREAD

**Issue by leeederek** [kurtosis-tech/kurtosis#924]
### Background & motivation

I have to remember what the localhost port is to use the kurtosis frontend.

### Desired behaviour

 I'd like to have the URL printed for me when a run of Kurtosis completes, or even better, auto-open the frontend in my browser.

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

**Comment by mieubrisse** [comment]
`kurtosis web` is now available to do this automatically

**Comment by mieubrisse** [comment]
@leeederek can we close this?

**Comment by leeederek** [comment]
No, the addition of `kurtosis web` does not solve my pain point.

My pain point is that it is not intuitive to discover the kurtosis frontend interface. A user would need to *know that the kurtosis web command exists and to use it*. 

I would much prefer that at the end of each run, the automatic `kurtosis enclave inspect` output shows me a way to navigate to the frontend with a button click
